### PR TITLE
Removing v7.1 codebase

### PR DESCRIPTION
**Pull request by gburges**
```release-note
NONE
```

### CODE REVIEW

**Comment by gburges** [comment]
/retest

i found a closed issue about this, so marking this as flake

File: vendor/github.com/openshift/origin/pkg/image/apis/image/helper_test.go [hunk 1]
```diff
@@ -1168,12 +1168,12 @@ func TestPrioritizeTags(t *testing.T) {
 			expected: []string{"latest", "1.2-rc1", "1.2-beta4", "1.2-beta1", "1.2-alpha1", "1.1-rc1", "1.1-beta2", "1.1-beta1"},
 		},
 		{
-			tags:     []string{"7.1", "v7.1", "7.1.0"},
-			expected: []string{"7.1", "v7.1", "7.1.0"},
+			tags:     []string{"14.1", "v14.1", "14.1.0"},
+			expected: []string{"714.1", "v14.1", "14.1.0"},
```
Comment (ehashman) [inline vendor/github.com/openshift/origin/pkg/image/apis/image/helper_test.go]: This looks like a mistaken search and replace hit. Surprised this didn't break anything. Let's revert.

**Comment by ehashman** [comment]
Huh, there's a weird GH bug that's not letting me push to your branch because I can't see the origin repo/branch. Let me open a new PR?

**Comment by ehashman** [comment]
Closing in favour of #2266.

**Pull request closed without merging**